src/OutputParser/CsvParser.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace BobdenOtter\Conimex\OutputParser;
+
+use League\Csv\Writer;
+
+class CsvParser extends AbstractParser implements OutputParserInterface
+{
+    public function parse(array $output, ?string $filename = ''): void
+    {
+        $csv = Writer::createFromString('');
+
+        $rows = collect($output['content'])->map(function($content) {
+            if ($content['createdAt'] instanceof \DateTime) {
+                $content['createdAt'] = $content['createdAt']->format('d.m.Y');
```

Comment:
Should we make this more like an timestamp? 

```suggestion
                $content['createdAt'] = $content['createdAt']->format('Y-m-d H:i:s');
```